src/Api/Managers/NotificationManager.cs

```diff
@@ -1,65 +1,87 @@
-﻿using BLL.Managers;
-using BOL.Data;
+﻿using BOL.Data;
 using BOL.Models;
+using Microsoft.EntityFrameworkCore;
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using System.Threading.Tasks;
 
 namespace BLL.Managers
 {
     public class NotificationManager : BaseManager
     {
         public NotificationManager(CmsDbContext dbContext) : base(dbContext) { }
 
-        public virtual IEnumerable<Notification> CurrentUser(User userId)
+        public virtual IEnumerable<List<Notification>> CurrentUser(User userId)
         {
             return (from n in dbContext.Notifications
```

Comment:
Hm why do you query the Users Table?
I would expect something like this:
return dbContext.Notifications.Where(n => n.userId == userId).OrderBy(n => n.timeStamp).toList();